core/templates/components/ck-editor-helpers/customize-rte-component-modal.template.html

```diff
@@ -35,6 +35,7 @@ <h3>Customize This Component</h3>
 <style>
   .oppia-customization-arg-form-container .oppia-customization-arg-editor {
     margin-bottom: 25px;
+    z-index: 3001;
```

Comment:
Do we need to change this or change the z-index value around line 49? Also, have you tested the changes in the exploration editor page?

Reply:
Also, the changes in line 49 (z-index: 1) were introduced via https://github.com/oppia/oppia/pull/17489 can you please ensure that the new changes don't introduce regression (#15935).

Reply:
Can you please try adding a concept card in the hint modal i.e, try adding a hint with a concept card? is it working fine?

Reply:
I tested every single thing and everything is working fine there doesn't exist any issue now.
here you can see it

[Screencast from 04-28-2023 10:00:22 AM.webm](https://user-images.githubusercontent.com/118007662/235054608-0c688bc7-1de0-4d91-b454-2ad5aeaeae7c.webm)

Reply:
@SanjaySajuJacob Can you please look at this thread and help us test and check whether we need z-index in both places?